pkg/commands/backend/update.go

```diff
@@ -236,6 +247,36 @@ func (c *UpdateCommand) Exec(_ io.Reader, out io.Writer) error {
 		input.SSLCiphers = &c.SSLCiphers.Value
 	}
 
+	if c.TCPKaEnable.WasSet {
+		var tcpKaEnable bool
+
+		switch c.TCPKaEnable.Value {
+		case "true":
+			tcpKaEnable = true
+		case "false":
+			tcpKaEnable = false
+		default:
+			err := errors.New("'tcp-ka-enable' flag must be one of the following [true, false]")
+			c.Globals.ErrLog.Add(err)
+			return err
```

Comment:
Same comment as for `describe`; the unit test should exercise all three paths in this switch statement.

Reply:
Done in 208e61da518bc6cd44cf0a7563eb0449b1aff2fb!